src/ui/map.js

```diff
@@ -1285,8 +1329,9 @@ class Map extends Camera {
             }
         };
 
-        if (this._delegatedListeners && this._delegatedListeners[type]) {
-            removeDelegatedListener(this._delegatedListeners);
+        const deletegatedListeners = this._delegatedListeners ? this._delegatedListeners[type] : undefined;
+        if (deletegatedListeners) {
+            removeDelegatedListener(deletegatedListeners);
```

Comment:
nit: since it's necessarily plural, and to fix a typo:
```suggestion
            removeDelegatedListeners(delegatedListeners);
```